Resources/views/Configuration/config.js.twig

```diff
@@ -44,6 +44,8 @@
                 .icon('<span class="glyphicon glyphicon-{{ entity.icon }}"></span>');
 
             {{ entity.name }}.dashboardView()
+                .title('Recent {{ entity.name }}')
```

Comment:
Probably over the weekend, I have limited availability until then unfortunately.